src/sidebar/components/Annotation/AnnotationBody.js

```diff
@@ -18,6 +18,34 @@ import TagListItem from '../TagListItem';
  * @typedef {import("../../../types/config").SidebarSettings} SidebarSettings
  */
 
+/**
+ * Button to expand or collapse the annotation excerpt (content)
+ * @param {object} props
+ *   @param {string} [props.classes]
+ *   @param {(collapse:boolean) => void} props.setIsCollapsed
+ *   @param {boolean} props.isCollapsed
+ */
+function ToggleExcerptButton({ classes, setIsCollapsed, isCollapsed }) {
+  const toggleText = isCollapsed ? 'More' : 'Less';
+  return (
+    <LabeledButton
+      classes={classnames('text-grey-7 font-normal', classes)}
+      expanded={!isCollapsed}
+      onClick={() => setIsCollapsed(!isCollapsed)}
+      title={`Toggle visibility of full annotation text: Show ${toggleText}`}
+    >
+      <div className="flex items-center gap-x-2">
+        <Icon
+          classes="!text-tiny"
+          name={isCollapsed ? 'expand' : 'collapse'}
+          title={isCollapsed ? 'expand' : 'collapse'}
+        />
+        <div>{toggleText}</div>
```

Comment:
What is the unstyled `<div>` for here?

Reply:
It's a container to explicitly establish a `flex` item here, a sibling to the `Icon` component's element. It is not strictly necessary so I can remove it if you prefer.